ndc-models/src/lib.rs

```diff
@@ -689,9 +717,25 @@ pub struct RowSet {
     pub aggregates: Option<IndexMap<String, serde_json::Value>>,
     /// The rows returned by the query, corresponding to the query's fields
     pub rows: Option<Vec<IndexMap<String, RowFieldValue>>>,
+    ///
+    pub groups: Option<Vec<Group>>,
 }
 // ANCHOR_END: RowSet
 
+// ANCHOR: Group
+#[skip_serializing_none]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, JsonSchema)]
+#[schemars(title = "Group")]
+pub struct Group {
+    /// 
+    pub dimensions: IndexMap<String, serde_json::Value>,
+    ///
+    pub aggregates: IndexMap<String, serde_json::Value>,
+    ///
+    pub rows: Vec<IndexMap<String, RowFieldValue>>,
```

Comment:
We removed this from scope. You can't return rows of groups in the current Aggregates/Grouping RFC.